install/threesdk/InstallTools.py

```diff
@@ -1882,6 +1883,20 @@ def _installbase_for_shell():
                 sudoremove = False
             Tools.execute(script, interactive=True, sudo_remove=sudoremove)
 
+    @staticmethod
+    def get_newest_version():
+        from threesdk import __version__
+        # call releases api
+        resp = requests.get('https://api.github.com/repos/threefoldtech/jumpscaleX_core/releases') 
```

Comment:
bare in mind this result might be paginated...
you need to make sure you collect all results (see github api)

Reply:
if you are only interested in the latest you can use https://api.github.com/repos/threefoldtech/jumpscaleX_core/releases/latest

Reply:
updated API call to the latest.